### ISSUE TITLE
Help me

### ISSUE THREAD

**Issue by HanNano** [jellyfin/jellyfin#10126]
jellyfin.service
     Loaded: bad-setting (Reason: Unit jellyfin.service has a bad unit file setting.)
    Drop-In: /etc/systemd/system/jellyfin.service.d
             └─jellyfin.service.conf
     Active: inactive (dead)

Aug 19 08:28:42 Jellyno systemd[1]: jellyfin.service: Service has no ExecStart=, ExecStop=, or SuccessAction=. Refusing.
Aug 19 08:45:33 Jellyno systemd[1]: jellyfin.service: Service has no ExecStart=, ExecStop=, or SuccessAction=. Refusing.
Aug 19 08:45:56 Jellyno systemd[1]: jellyfin.service: Service has no ExecStart=, ExecStop=, or SuccessAction=. Refusing.
Aug 19 08:49:39 Jellyno systemd[1]: jellyfin.service: Service has no ExecStart=, ExecStop=, or SuccessAction=. Refusing.
Aug 19 08:49:39 Jellyno systemd[1]: jellyfin.service: Service has no ExecStart=, ExecStop=, or SuccessAction=. Refusing.
Aug 19 08:49:39 Jellyno systemd[1]: jellyfin.service: Service has no ExecStart=, ExecStop=, or SuccessAction=. Refusing.
Aug 19 08:51:09 Jellyno systemd[1]: jellyfin.service: Service has no ExecStart=, ExecStop=, or SuccessAction=. Refusing.
Aug 19 09:04:09 Jellyno systemd[1]: jellyfin.service: Service has no ExecStart=, ExecStop=, or SuccessAction=. Refusing.
Aug 19 09:04:09 Jellyno systemd[1]: jellyfin.service: Service has no ExecStart=, ExecStop=, or SuccessAction=. Refusing.
Aug 19 09:04:09 Jellyno systemd[1]: jellyfin.service: Service has no ExecStart=, ExecStop=, or SuccessAction=. Refusing.
Warning: The unit file, source configuration file or drop-ins of jellyfin.service changed on disk. Run 'systemctl daemon-reload' to reload units.
dpkg: error processing package jellyfin-server (--configure):
 installed jellyfin-server package post-installation script subprocess returned error exit status 1
dpkg: dependency problems prevent configuration of jellyfin:
 jellyfin depends on jellyfin-server; however:
  Package jellyfin-server is not configured yet.

dpkg: error processing package jellyfin (--configure):
 dependency problems - leaving unconfigured
No apport report written because the error message indicates its a followup error from a previous failure.
                          Processing triggers for libc-bin (2.35-0ubuntu3.1) ...
Errors were encountered while processing:
 jellyfin-server
 jellyfin
E: Sub-process /usr/bin/dpkg returned an error code (1)

**Comment by Donien** [comment]
### Please try to better lay out your specific problem in the future. If you only copy-paste some random error message, most people won't be willing to actually help you, even if they could.


This is (most definitely) no problem with Jellyfin.  
It is a problem with **your** installation.

And this is nothing that should be here as an issue unless it is reproducible regardless of your specific system.  


Even though this should not be here, you might find this helpful: https://www.makeuseof.com/how-to-find-and-fix-broken-packages-on-linux/

TLDR - try running:

```
sudo apt --fix-missing update
sudo apt update
sudo apt install -f

sudo dpkg --configure -a 
```

Good luck.

**Comment by cvium** [comment]
What Donien said. If you require more help, I suggest you visit our forums or our troubleshooting room on Matrix https://jellyfin.org/contact
